ili/dataloaders/loaders.py

```diff
@@ -475,5 +476,71 @@ def load_parameters(
         ).iloc[nodes]
         return theta[param_names].values
 
+
+class TorchLoader(_BaseLoader):
```

Comment:
Please add unit tests to ensure that all functions in this class return what is as expected (as we do for e.g. the numpy loader)

Reply:
Done.